src/main/java/de/doubleslash/keeptime/view/ManageWorkController.java

```diff
@@ -110,7 +110,7 @@ private void initialize() {
 
       setProjectUpComboBox();
 
-      Platform.runLater(() -> projectComboBox.requestFocus());
+      Platform.runLater(() -> noteTextArea.requestFocus());
```

Comment:
Was not requested: But would it be possible to move the cursor to the end of the note? may be nice :)

Reply:
Cursor is now at the end of the Text 👍